### PR TITLE
Visualization tutorial using Earth Engine Python API, geemap and OpenAltimetry

### PR DESCRIPTION
**Pull request by fliphilipp**
Adding this as a draft.

Preview at https://deploy-preview-74--icesat2-2022.netlify.app/tutorials/datavisualization/visualization_earth_engine_geemap_is2_hw_2022

### CODE REVIEW

**Comment by scottyhq** [comment]
thanks @fliphilipp can you please add a link to your notebook here? https://github.com/ICESAT-2HackWeek/website2022/blob/acd8d3ff2d655ac0da0c10cd11460c70b41af8d9/book/_toc.yml#L26-L30 

since we don't want to execute this via CI (since it requires interactive login) you can add it to our 'excluded notebooks' here https://github.com/ICESAT-2HackWeek/website2022/blob/acd8d3ff2d655ac0da0c10cd11460c70b41af8d9/book/_config.yml#L37-L38

Is the notebook still a draft or do you consider it ready to merge?

**Comment by fliphilipp** [comment]
Ok I'll add it to the toc and excluded notebooks. This is still a draft.

**Comment by fliphilipp** [comment]
This should be ready to be merged!

File: book/_config.yml [hunk 1]
```diff
@@ -38,6 +38,8 @@ execute:
     - "**/dataintegration-1.ipynb"
     - "**/dataintegration-2.ipynb"
     - "**/geospatial-advanced.ipynb"
+    - "**/Visualization_Earth_Engine_geemap_IS2_HW_2022.ipynb"
+    - "**Visualization_geemap_use_examples.ipynb"
```
Comment (weiji14) [inline book/_config.yml:42]: Missing slash causing the builds to break.
```suggestion
    - "**/Visualization_geemap_use_examples.ipynb"
```

**Review by weiji14 — COMMENTED** [review]
Great work @fliphilipp! Really love those cross-section plots :smile:

I've just got two minor suggestions:
1. Add the missing slash to fix the tests
2. Maybe replace the 11MB GIF with just a link. Specifically, delete the `misc/oa_find_data.gif` file, and change this line in Visualization_Earth_Engine_geemap_IS2_HW_2022.ipynb:
```
<img src="misc/oa_find_data.gif" alt="gif showing how to get to the data in OpenAltimetry" width="600px">
```
to
```
![gif showing how to get to the data in OpenAltimetry](https://user-images.githubusercontent.com/23487320/159029794-748ff020-32a7-4794-958d-d27340ffd6ca.gif)
```
This is just a compressed ~10MB version of your GIF I've uploaded to GitHub.

gif showing how to get to the data in OpenAltimetry

Will help to reduce the size of the git repository, so people doing `git clone` or `git pull` don't download so much data.

**Review by weiji14 — APPROVED** [review]
Cool, just need to remember to do "Squash and merge" for this (instead of a normal merge).

**Review by scottyhq — APPROVED** [review]
Truly fantastic notebook @fliphilipp! Thank you.

**Pull request merged**